### ISSUE TITLE
[Accessibility Bug] In 400% zoom, content present in grid is not visible completely

### ISSUE THREAD

**Issue by Kitwradr** [telerik/kendo-angular#3540]
**Describe the bug**

When we zoom the grid to 400% the pagination controls are disappearing (the dropdown for selecting the number of items)

WCAG Reference: 

https://www.w3.org/WAI/WCAG21/Understanding/reflow.html 

**To Reproduce**

Zoom to 400% in this demo site : https://www.telerik.com/kendo-angular-ui/components/grid/examples/configuration/paging/pager-settings/?theme=default-main&themeVersion=4.42.0

**Expected behavior**
n 400% zoom mode, Content present in the page should be clearly visible

**Screenshots**
![image](https://user-images.githubusercontent.com/26614919/148360815-a86ff2a9-28c9-42c6-abd0-e0d1b2e61d38.png)

**Browser**
 - OS: Windows
 - Browser Microsoft Edge

**Additional context**

**Comment by mbechev** [comment]
Hi @Kitwradr,

This is the default behavior of the Pager. When the Pager width is less than 768 pixels, some of its elements are hidden. For more details please check this article:

https://www.telerik.com/kendo-angular-ui/components/grid/paging/#toc-responsive-pager

To disable this feature set the **responsive** option of the [pageable ](https://www.telerik.com/kendo-angular-ui/components/grid/api/GridComponent/#toc-pageable)property to false:

```ts
 <kendo-grid
            [data]="gridView"
            [pageable]="{responsive: false}"
            ...
>
        ...
</kendo-grid>
```
